packages/material-ui/test/integration/Select.test.js

```diff
@@ -39,40 +40,59 @@ describe('<Select> integration', () => {
       );
     }
 
-    it('should focus the selected item', async () => {
+    /**
+     * @type {ReturnType<typeof useFakeTimers>}
+     */
+    let clock;
+    beforeEach(() => {
+      clock = useFakeTimers();
+    });
+
+    afterEach(() => {
+      clock.restore();
+    });
+
+    it('should focus the selected item', () => {
       const { getByTestId, getAllByRole, getByRole, queryByRole } = render(<SelectAndDialog />);
+      expect(getByRole('button')).to.have.text('Ten');
```

Comment:
Should we share the same variable?
```suggestion
      expect(trigger).to.have.text('Ten');
```

Reply:
That isn't supposed to be there anyway.